requirements.md

```diff
@@ -0,0 +1,9 @@
+# FishSense Lite System Requirements
+- [ ] Maximum effective distance: 3 m
+- [ ] Minimum effective distance: 0.5 m? (driven)
```

Comment:
What does "driven" mean in this context?

Reply:
`driven` means that this is not a defining requirement, i.e. something else determines this requirement